docs/data-sources/apig_orchestration_rule_associated_apis.md

```diff
@@ -0,0 +1,85 @@
+---
+subcategory: "API Gateway (Dedicated APIG)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_apig_orchestration_rule_associated_apis"
+description: |-
+  Use this data source to get API list associated with the specified orchestration rule within HuaweiCloud.
+---
+
+# huaweicloud_apig_orchestration_rule_associated_apis
+
+Use this data source to get API list associated with the specified orchestration rule within HuaweiCloud.
+
+## Example Usage
+
+### Querying all APIs associated with the specified orchestration rule
+
+```hcl
+variable "instance_id" {}
+variable "orchestration_id" {}
+
+data "huaweicloud_apig_orchestration_rule_associated_apis" "test" {
+  instance_id      = var.instance_id
+  orchestration_id = var.orchestration_id
+}
+```
+
+### Querying API associated with the orchestration rule using specified API ID
+
+```hcl
+variable "instance_id" {}
+variable "orchestration_id" {}
+variable "api_id" {}
+
+data "huaweicloud_apig_orchestration_rule_associated_apis" "test" {
+  instance_id      = var.instance_id
+  orchestration_id = var.orchestration_id
+  api_id           = var.api_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `instance_id` - (Required, String) Specifies the ID of the dedicated instance to which the orchestration rule belongs.
+
+* `orchestration_id` - (Required, String) Specifies the ID of the orchestration rule.
```

Comment:
According to another data source `data.huaweicloud_apig_orchestration_rules`, this parameter names `rule_id` is better.